scripts/linux/mk-sdcard-img.sh

```diff
@@ -156,68 +142,43 @@ do
     loop_lp=`echo "${loop}" | cut -d ' ' -f3,3`
     loop_l=`echo "${loop}" | cut -d ' ' -f8,8`
 
-    export SDCARD_L_FILE_PATH="${loop_l}"
+    SDCARD_L_FILE_PATH="${loop_l}"
 
     if [ "${i}" -eq "0" ]; then
-        export SDCARD_P1_FILE_PATH="/dev/mapper/${loop_lp}"
+        SDCARD_P1_FILE_PATH="/dev/mapper/${loop_lp}"
     elif [ "${i}" -eq "1" ]; then
-        export SDCARD_P2_FILE_PATH="/dev/mapper/${loop_lp}"
-    elif [ "${i}" -eq "2" ]; then
-        export SDCARD_P3_FILE_PATH="/dev/mapper/${loop_lp}"
+        SDCARD_P2_FILE_PATH="/dev/mapper/${loop_lp}"
     fi
 done
 IFS=${oldIFS}
 echo "* Loop block device: ${SDCARD_L_FILE_PATH}"
 echo "* Loop block p1 device: ${SDCARD_P1_FILE_PATH}"
 echo "* Loop block p2 device: ${SDCARD_P2_FILE_PATH}"
-echo "* Loop block p3 device: ${SDCARD_P3_FILE_PATH}"
 
 echo "Formatting partitions..."
 mkfs.vfat -n "BOOT" "${SDCARD_P1_FILE_PATH}" > /dev/null
-mkfs.ext4 -L "RecalBox OS" "${SDCARD_P2_FILE_PATH}" > /dev/null
-mkfs.ext4 -L "RecalBox Share" "${SDCARD_P3_FILE_PATH}" > /dev/null
+mkfs.ext4 -L "RECALBOX" "${SDCARD_P2_FILE_PATH}" > /dev/null
 tune2fs -m 0 "${SDCARD_P2_FILE_PATH}"
-tune2fs -m 0 "${SDCARD_P3_FILE_PATH}"
 
 echo "Mounting partitions..."
 mount -o loop "${SDCARD_P1_FILE_PATH}" "${SDCARD_BOOT_PWD}"
 mount -o loop "${SDCARD_P2_FILE_PATH}" "${SDCARD_ROOT_PWD}"
-mount -o loop "${SDCARD_P3_FILE_PATH}" "${SDCARD_SHARE_PWD}"
 
 echo "Writing partitions..."
 tar --no-same-permissions --no-same-owner -xf "${BR_IMAGES_PWD}/recalbox/boot.tar.xz" -C "${SDCARD_BOOT_PWD}"
 tar -xf "${BR_IMAGES_PWD}/recalbox/root.tar.xz" -C "${SDCARD_ROOT_PWD}"
-tar -xf "${BR_IMAGES_PWD}/recalbox/share.tar.xz" -C "${SDCARD_SHARE_PWD}"
-#sed -i -e 's,ext2,ext4,' -e 's,\(/dev/mmcblk0p3\),#\1,' "${SDCARD_ROOT_PWD}/etc/fstab"
-#sed -i -e 's,#\(si1\),\1,' "${SDCARD_ROOT_PWD}/etc/inittab"
 
 echo "Syncing and unmounting partitions..."
 sync
 umount "${SDCARD_BOOT_PWD}"
 umount "${SDCARD_ROOT_PWD}"
-umount "${SDCARD_SHARE_PWD}"
 
 echo "Destroying block devices..."
 kpartx -sd "${SDCARD_IMG_FILE_PATH}" > /dev/null
 
 echo "Cleaning..."
 rm -fr "${SDCARD_PWD}"
-
-echo "Moving to testing area..."
-baseName=`basename "${SDCARD_IMG_FILE_PATH}" .img`
-releaseHome="${HOME}"
-if [ -d "/home/recalbox" ]; then
-    releaseHome="/home/recalbox"
-fi
-releaseUser="`basename \"${releaseHome}\"`"
-releaseFilePath="${releaseHome}/public_html/${baseName}.tar.xz"
-mkdir -p "`dirname \"${releaseFilePath}\"`"
-chown -R ${releaseUser}:www-data "${SDCARD_IMG_FILE_PATH}"
-chmod -R u+rwx,g+rwx,o+rx "${SDCARD_IMG_FILE_PATH}"
-tar -cJvf "${releaseFilePath}" -C "`dirname \"${SDCARD_IMG_FILE_PATH}\"`" "`basename \"${SDCARD_IMG_FILE_PATH}\"`" > /dev/null
-chown -R ${releaseUser}:www-data "${releaseFilePath}"
-chmod -R u+rwx,g+rwx,o+rx "${releaseFilePath}"
-rm "${SDCARD_IMG_FILE_PATH}"
```

Comment:
Move that to an unused function instead of removing it - i do want to use it on my server